src/script/components/calling/CallingCell.tsx

```diff
@@ -51,26 +51,31 @@ import {ContextMenuEntry, showContextMenu} from '../../ui/ContextMenu';
 import {initFadingScrollbar} from '../../ui/fadingScrollbar';
 import {CallActions, CallViewTab} from '../../view_model/CallingViewModel';
 
-export interface CallingCellProps {
+interface VideoCallProps {
+  multitasking: Multitasking;
+  hasAccessToCamera?: boolean;
+  isSelfVerified?: boolean;
+  teamState?: TeamState;
+}
```

Comment:
Those are the props that are not used by the simpler answering controls component. Is there a better way than making them conditional?